test/test_argparse.py

```diff
@@ -75,7 +75,7 @@ def test_s(self):
             "function a str is required"
         )
 
-    def test_B(self):
+    def test_upper_b(self):
```

Comment:
PyPy uses an older version of pytest that does not support upper case test names, so only one of `test_b`, `test_B` ran

Reply:
A bit strange, but okay. :)